server/api/auth/localStrategyVerifyCallback.ts

```diff
@@ -18,11 +19,18 @@ export const localStrategyVerifyCallback = async (req: Request, email: string, p
       if (invitationUuid) {
         const { user: invitedUser } = await UserController.readByInvitation({ invitationUuid })
         if (invitedUser?.status !== 'active') {
-          // TODO add password into userprovider
+          const provider =  {
+            userId: invitedUser.id,
+            provider: AuthProvider.local,
+            props: {
+              password: await bcrypt.hash(password, 10)
+            }
+          }
+          await UserProviderRepository.create({ provider })
```

Comment:
Should we use controller instead to be consistent?

Reply:
let's skip this for now. it's the provider will never be used elsewhere for now.
what do you think ?

Reply:
yes @minotogna , we discussed but I did it because it will be useful for "reset password" issue.